webrtc-stats/getStats-remote-candidate-address.html

```diff
@@ -0,0 +1,77 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>Exposure or remote candidate address on stats</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="../webrtc/RTCPeerConnection-helper.js"></script>
+<script src="../webrtc/RTCStats-helper.js"></script>
+<script>
+  'use strict';
+
+promise_test(async () => {
+    const localPc = new RTCPeerConnection();
+    const remotePc = new RTCPeerConnection();
+
+    const promiseDataChannel = new Promise(resolve => {
+        remotePc.addEventListener('datachannel', (event) => {
+            resolve(event.channel);
+        });
+    });
+
+    const localDataChannel = localPc.createDataChannel('test');
+
+    localPc.addEventListener('icecandidate', event => {
+        if (event.candidate)
+            remotePc.addIceCandidate(event.candidate);
+    });
+    doSignalingHandshake(localPc, remotePc);
+
+    const remoteDataChannel = await promiseDataChannel;
+
+    assert_true(!!remoteDataChannel, "remoteDataChannel is defined");
+    await new Promise(resolve => {
+        remoteDataChannel.onmessage = resolve;
+        localDataChannel.send("test");
+    });
+
+    const remoteCandidateStats = getRequiredStats(await localPc.getStats(), "remote-candidate");
+    assert_equals(remoteCandidateStats.address, null, "address should be null");
```

Comment:
[address](https://w3c.github.io/webrtc-stats/#dom-rtcicecandidatestats-address) isn't nullable. This should be `undefined` according to the name of this PR, right?

Reply:
I changed the name of the PR since it was discussed in webrtc-pc to go with null instead of undefined for this case.